### ISSUE TITLE
How can I know which of my variables was most important to generate the model?

### ISSUE THREAD

**Issue by Oliveira-D-R** [wallaceEcoMod/wallace#353]
I've seen this information available in several niche modeling surveys. However, at Wallace I don't know how to display this information.

**Comment by gepinillab** [comment]
Hi @Oliveira-D-R,

This option is not available on Wallace. For maxent, `Wallace` depends on the `dismo` or the `maxnet` package. As far as I know, neither of them returns the information of the most important variables (so we cannot incorporate them on Wallace).

Best,
Gonzalo

**Comment by jamiemkass** [comment]
@Oliveira-D-R, actually the Maxent object in `dismo` does have the information for the "percent contribution" and "permutation importance" values in the `@results` slot. Be advised that the "percent contribution" changes with every run, while the "permutation importance" is deterministic. See the "Brief Tutorial for Maxent" by Steven Phillips, which explains this. 

At the present, we do not include this info in _Wallace_, but we may in the future. For now, you can download the session code, generate your models in R, then get the variable importance values from the model object.

**Comment by gepinillab** [comment]
Good point @jamiemkass. I completely forgot about it the `dismo` option. I will add this option as a someday enhancement.

**Comment by dlema82** [comment]
Hello,
I have done as suggested, running the models from the wallace session code, but my model object does not seem to have these data ( percent contribution and permutation importance) in the @results partition. I have looked as well for these data in each of the @models partitions.

Is there a code that I can use to extract or calculate these from the model object?
